.gitmodules

```diff
@@ -3,94 +3,78 @@
 	url = https://github.com/nestlabs/nlassert.git
 	branch = master
 	ignore = dirty
-	commit = bd2f082102d4456ca750189ee853fa8c7443af59
```

Comment:
Why these various changes to pull tip of master instead of specific commits?  That doesn't seem right...

Reply:
These lines were only used by the automake build for its initial clones (since it didn't depend on git). Submodules stores the commit pointers inline in the git trees.

I've been keeping these two sources of truth in sync but we can remove them now; doing so has no effect on the submodule workflow.

Reply:
Ah, gotcha.